lib/engine-addon.js

```diff
@@ -387,10 +391,76 @@ module.exports = {
         return mergeTrees([
           externalTree,
           engineJSTree,
-          engineCSSTree
+          compiledEngineCSSTree
         ].filter(Boolean), { overwrite: true });
       };
 
+      this.compileLazyEngineStyles = function compileLazyEngineStyles(vendorTree, externalTree) {
+        var vendorCSSTree = buildVendorCSSTree.call(this, vendorTree);
+        var engineStylesOutputDir = 'engines-dist/' + this.name + '/assets/';
```

Comment:
I'd prefer `path.join` unless `preprocessCss` expects POSIX in which case `ensure-posix-path`. We currently have to know whether we've included leading/trailing slashes anywhere we modify it such as https://github.com/ember-engines/ember-engines/pull/324/files#diff-1c2c32318ef6ba8d9be4e7b085f8ecb5R419